nservicebus/eventstoretransport/outbox_diagram_eventstore_[,2).partial.md

```diff
@@ -0,0 +1,26 @@
+```
```

Comment:
@SzymonPobiega shouldn't be name of this file `index_topology_eventstoretransport_[2,).partial.md`